river/Config.zig

```diff
@@ -171,6 +175,30 @@ pub fn shouldFloat(self: Self, view: *View) bool {
     return false;
 }
 
+pub fn getSpawnTagsFilter(self: Self, view: *View) u32 {
```

Comment:
```suggestion
pub fn getSpawnTagsFilter(self: Self, view: *View) ?u32 {
```

Return tags if found, `null` if not.